src/shadertools/modules/fp64/test/fp64-shader.spec.js

```diff
@@ -251,9 +250,9 @@ function test_float_add(gl, testName) {
   const program = new Program(gl, assembleShaders(gl, {
         vs: require('./vs_float_add.glsl'),
         fs: require('./fs.glsl'),
-        fp64: true,
-        project64: true
-      }));
+        modules: ['fp64']
```

Comment:
Hmm I thought `modules` was only supported by `Model` not` Program`. But if this works, then of course fine!